pkg/render/manager.go

```diff
@@ -495,13 +513,19 @@ func (c *managerComponent) managerContainer() corev1.Container {
 func (c *managerComponent) managerOAuth2EnvVars() []v1.EnvVar {
 	var envs []corev1.EnvVar
 
-	if c.dexCfg == nil {
+	if c.keyValidatorConfig == nil {
 		envs = []corev1.EnvVar{{Name: "CNX_WEB_AUTHENTICATION_TYPE", Value: "Token"}}
 	} else {
 		envs = []corev1.EnvVar{
 			{Name: "CNX_WEB_AUTHENTICATION_TYPE", Value: "OIDC"},
-			{Name: "CNX_WEB_OIDC_AUTHORITY", Value: fmt.Sprintf("%s/dex", c.dexCfg.ManagerURI())},
-			{Name: "CNX_WEB_OIDC_CLIENT_ID", Value: DexClientId}}
+			{Name: "CNX_WEB_OIDC_CLIENT_ID", Value: c.keyValidatorConfig.ClientID()}}
+
+		switch c.keyValidatorConfig.(type) {
+		case *DexKeyValidatorConfig:
+			envs = append(envs, corev1.EnvVar{Name: "CNX_WEB_OIDC_AUTHORITY", Value: c.keyValidatorConfig.Issuer()})
+		case *tigerakvc.KeyValidatorConfig:
+			envs = append(envs, corev1.EnvVar{Name: "CNX_WEB_OIDC_AUTHORITY", Value: ""})
+		}
```

Comment:
I think this could be pretty easy to whip up a test for if you think it'd be valuable, given what you told me about how easy it is in go to accidentally match a struct with the wrong interface if one interface is a subset of another.

the test would create a managerComponent with each keyValidatorConfig, then call `managerOauth2EnvVars()` and assert that `CNX_WEB_OIDC_AUTHORITY` is called correctly